src/scenes/lumbridge.js

```diff
@@ -19,13 +19,15 @@ export class LumbridgeScene extends EnemyLevelScene {
                     name: "cow",
                     path: "src/assets/sprites/Cow.png",
                     maxHealth: 8,
-                    killGold: 5
+                    killGold: 5,
+                    drops: [{item: "Bones", mat: "Normal", rate: .5}]
```

Comment:
Probably want to be consistent between mat and material

Reply:
I thought I changed this to material.